gitlens/gitlens-settings.md

```diff
@@ -265,6 +265,41 @@ GitLens is highly customizable and provides many configuration settings to allow
 <td><code>gitlens.views.showRelativeDateMarkers</code></td>
 <td>Specifies whether to show relative date markers (<em>Less than a week ago</em>, <em>Over a week ago</em>, <em>Over a month ago</em>, etc) on revision (commit) histories in the views</td>
 </tr>
+<td><code>gitlens.views.commits.files.icon</code></td>
```

Comment:
I think an opening `<tr>` is missing here (though it still rendered fine in vs code preview without it). The rest of the updates look good to me!